vtex/toolbelt
# Use new proprietary directory

#### What is the purpose of this pull request?
Now we will have our own `~/.vtex` directory :sparkles: 

#### How should this be manually tested?
```
cd /tmp && rm -rf toolbelt && \
git clone https://github.com/vtex/toolbelt.git && \
cd toolbelt && \
git checkout feat/use-new-directory && \
yarn && yarn watch
```
- Just login into an account, the new files should be on `~/.vtex/session
- Check on `~/vtex` if the folders `pretasks` and `telemetry` are created.
- Check if the folder `~/.config/configstore/vtex` is deleted.

#### Types of changes

#### Chores checklist
